### PR TITLE
Add RadarDataset class

### PR DESCRIPTION
**Pull request by abelsiqueira**
Add class `RadarDataset` to read a folder of radar files.

Python 3.11 is not supported, so I remove it as well.

### CODE REVIEW

File: bird_cloud_gnn/radar_dataset.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    checked. If the number of neighbours is big enough, the point is selected. A data point is
+    a graph created by taking a selected point and a number of its neighbours.
+
+
+    Attributes:
+        data_folder (str): Folder with the CSV files.
+        features (array of str): List of features expected to be present at every CSV file.
+        target (str): Target column. 0, 1 or missing expected.
+        max_distance (float): Maximum distance to look for neighbours.
+        min_neighbours (int): If a point has less than this amount of neighbours, if is ignored.
```
Comment (lyashevska) [inline bird_cloud_gnn/radar_dataset.py:29]:
```suggestion
        min_neighbours (int): If a point has less than this amount of neighbours, it is ignored.
```

File: bird_cloud_gnn/radar_dataset.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        target,
+        name="Radar",
+        max_distance=500.0,
+        min_neighbours=100,
+    ):
+        """Constructor
+
+        Args:
+            data_folder (str): Folder with the CSV files.
+            features (array of str): List of features expected to be present at every CSV file.
```
Comment (lyashevska) [inline bird_cloud_gnn/radar_dataset.py:46]:
```suggestion
            features (array of str): List of features expected to be present in every CSV file.
```

**Review by lyashevska — APPROVED** [review]
Thanks, looks great. There are a couple of typos.

**Comment by abelsiqueira** [comment]
Thanks for the review, @lyashevska, I accepted your suggestions.

**Pull request merged**